packages/content-format/src/nodes/EmbedNode.ts

```diff
@@ -6,11 +6,18 @@ export interface EmbedNode extends Element<typeof EmbedNode.TYPE> {
     uuid: string;
     url: string;
     oembed: OEmbed;
+    layout: `${EmbedNode.Layout}`;
```

Comment:
The new property has to be validated in the `validateEmbedNode` function below.

Reply:
Thanks, fixed: https://github.com/prezly/content-format-js/pull/25/commits/4b25f8b2b6b5bf31fb6e178ba8d0a3cf72782324